src/material/list/list.scss

```diff
@@ -305,6 +305,12 @@ mat-action-list {
 
 .mat-list-item-disabled {
   pointer-events: none;
+
+  // Since we can't use a color to indicate that the list
+  // item is disabled, we have to use opacity instead.
+  @include cdk-high-contrast {
+    opacity: 0.5;
```

Comment:
What does the contrast ratio end up being with this?

Reply:
I haven't measured it, but this isn't the first place where we indicate disabled elements by reducing the opacity to 0.5.